### PR TITLE
Windows build support

### PR DESCRIPTION
**Pull request by maxhora**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,34 @@ source:
...
 
 build:
-  skip: true  # [win]
-  number: 0
+  number: 1
+  skip: true  # [win and py27]
+  
+  features:
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
```
Comment (wesm) [inline recipe/meta.yaml]: Maybe make this match what's in gflags-feedstock? https://github.com/conda-forge/gflags-feedstock/blob/master/recipe/meta.yaml#L20

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,21 +10,34 @@ source:
...
 test:
+  requires:
+    - python 3.4.*  # [win and py34]
+    - python 3.5.*  # [win and py35]
   commands:
-    - test -f $PREFIX/bin/flatc
-    - test -f $PREFIX/include/flatbuffers/flatbuffers.h
+    - test -f $PREFIX/bin/flatc                              # [not win]
+    - test -f $PREFIX/include/flatbuffers/flatbuffers.h      # [not win]
+    - if not exist %PREFIX%\\Library\\bin\\flatc.exe exit 1  # [win]
```
Comment (wesm) [inline recipe/meta.yaml]: Can you also test for flatbuffers.h on Windows?

File: recipe/meta.yaml (hunk 1 above)
Comment (maxhora) [reply recipe/meta.yaml]: Sure, it seems, usage of "py>=35" is not good at all, because is not compatible with future versions of python and msvc.

**Comment by maxhora** [comment]
@wesm , have updated the changes. py27 (msvc2008) build is disabled. Compiler won't find source file to include cstdin, even if msinttypes package is installed. That might be resolve with patch, I think.

**Comment by wesm** [comment]
OK, no problem. Merging this as is. Thank you!

**Pull request merged**